blockchain/types/tx_internal_data_fee_delegated_chain_data_anchoring.go

```diff
@@ -316,20 +317,29 @@ func (t *TxInternalDataFeeDelegatedChainDataAnchoring) Execute(sender ContractRe
 }
 
 func (t *TxInternalDataFeeDelegatedChainDataAnchoring) MakeRPCOutput() map[string]interface{} {
+	decoded, err := DecodeAnchoringDataToJSON(t.Payload)
+	if err != nil {
+		logger.Error("decode anchor payload", "err", err)
+	}
 	return map[string]interface{}{
 		"typeInt":            t.Type(),
 		"type":               t.Type().String(),
 		"gas":                hexutil.Uint64(t.GasLimit),
 		"gasPrice":           (*hexutil.Big)(t.Price),
 		"nonce":              hexutil.Uint64(t.AccountNonce),
 		"input":              hexutil.Bytes(t.Payload),
+		"inputJSON":          decoded,
 		"signatures":         t.TxSignatures.ToJSON(),
 		"feePayer":           t.FeePayer,
 		"feePayerSignatures": t.FeePayerSignatures.ToJSON(),
 	}
 }
 
 func (t *TxInternalDataFeeDelegatedChainDataAnchoring) MarshalJSON() ([]byte, error) {
+	decoded, err := DecodeAnchoringDataToJSON(t.Payload)
+	if err != nil {
+		logger.Error("decode anchor payload", "err", err)
```

Comment:
Is this okay not to return the error?

Reply:
@KimKyungup All types anchoring data should be unmarshalled with `DecodeAnchoringDataToJSON`. So, we can return an error if it failed, right?

Reply:
User can write any data to the payload. So decoding may be failed. 
So I think, I wrote this code not to return an error.

Reply:
@jeongkyun-oh @KimKyungup I will change the log level to `TRACE` in another PR.